config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -388,3 +388,36 @@ periodics:
     testgrid-tab-name: kubelet-serial-gce-e2e-graceful-node-shutdown
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: Executes graceful node shutdown e2e node tests
+
+- name: ci-kubernetes-node-kubelet-node-swap
+  interval: 4h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
+        args:
+          - --repo=k8s.io/kubernetes=master
+          - --timeout=240
+          - --root=/go/src
+          - --scenario=kubernetes_e2e
+          - --
+          - --deployment=node
+          - --gcp-project-type=node-e2e-project
+          - --gcp-zone=us-west1-b
+          - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-swap.yaml
+          - --node-test-args=--feature-gates=DynamicKubeletConfig=true --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --fail-swap-on=false"
+          - --node-tests=true
+          - --provider=gce
+          - --test_args=--nodes=1 --focus="\[NodeAlphaFeature:NodeMemorySwap\]"
```

Comment:
I've been avoiding using new NodeAlphaFeature tags where possible.

I think the focus of this test should be to run the full node e2e suite with swap on, per https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2400-node-swap/README.md#test-plan